HEN_HOUSE/user_codes/g/g.mortran

```diff
@@ -1203,6 +1228,31 @@ $GET_INPUT(IVAL);
 IF( error_flag = 0 ) [ calc_type = value(ival,1); ]
 ELSE                 [ error_flag = 0; ]
 
+"-------------------------------------------------"
+" Ignore sub-threshold relaxations?               "
+"-------------------------------------------------"
+ival                = ival + 1;
+values_sought(ival) = 'CLASSIFY SUB-THRESHOLD RELAXATIONS';
+nvalue(ival)        = 1;
+type(ival)          = 3;
+allowed_inputs(ival,0) = 'NO';
+allowed_inputs(ival,1) = 'YES';
+$GET_INPUT(IVAL);
+IF( error_flag = 0 ) [
```

Comment:
Ideally, we would harmonize the `IF` bracketing style. My perspective is that we ought to settle on conditional branch style with spaces around ` (...) ` (no spaces inside the brackets, and one space before the opening ` [`

```fortran
IF (...) [
  ...
]
ELSE (...) [
  ...
]
```